.changeset/four-buckets-brake.md

```diff
@@ -0,0 +1,5 @@
+---
+"comet-site-v8": major
```

Comment:
why is this a major change? shouldnt this be a minor?

Reply:
Version was changed.

354fedcc8ccf928fc50156e971b71c5532a4f8ff

Reply:
Changed charset doc 

94403b43fb7b9e93122391a8b503fc7b2f7cc63e